apps/srt-tunnel.cpp

```diff
@@ -478,6 +490,12 @@ class TcpMedium: public Medium
     // Just models. No options are predicted for now.
     void ConfigurePre(int )
     {
+        int optval = 1;
+#if defined(LINUX)
+        setsockopt(m_socket, SOL_SOCKET, MSG_NOSIGNAL, (const char*)&optval, sizeof(optval));
```

Comment:
`MSG_NOSIGNAL` is not a flag settable by `setsockopt`. It's a flag that should be passed in `send` function in the last argument.

Reply:
https://stackoverflow.com/questions/26752649/so-nosigpipe-was-not-decleared

It is also recommended that if this is simply a standalone application (not a library), you can simply disable the SIGPIPE signal.